lib/versioning/regex/index.ts

```diff
@@ -98,10 +98,6 @@ export class RegExpVersioningApi extends GenericVersioningApi<RegExpVersion> {
     );
   }
 
-  isStable(version: string): boolean {
```

Comment:
What's the implication removing this? Is it checked somewhere else instead?

Reply:
Yes, it's same implementation as in base class.